content/blog/trail-of-bits-security-review.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Trail Of Bits Security Review"
+description = "A security review of CoreDNS conducted by Trail of Bits"
+tags = ["Security Review", "Threat Model", "Trail of Bits"]
+date = "2022-02-24T00:00:00-00:00"
+author = "chris"
++++
+
+Trail of Bits (https://trailofbits.com) conducted a security review and threat model of CoreDNS.
```

Comment:
... and?

this post is a bunch of links, can we at least paraphrase things a bit